packages/ui/src/DetailInfo/DetailInfo.tsx

```diff
@@ -1,66 +1,99 @@
+import ActionInfo, { ActionInfoProps } from '@ui-kit/shared/ui/ActionInfo'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { ReleaseChannel } from '@ui-kit/utils'
 import Box from 'ui/src/Box/Box'
 import Loader from 'ui/src/Loader/Loader'
 
+const { Spacing } = SizesAndSpaces
+
 type Variant = 'error' | 'warning' | 'success' | ''
-type Size = 'xs' | 'sm' | 'md' | 'lg'
+
+/**
+ * Maps the `variant` prop to the corresponding `valueColor` prop for the `ActionInfo` component.
+ * This should be removed when we get rid of this file and use `ActionInfo` directly.
+ * */
+const VariantToColorMap = {
+  '': 'textPrimary',
+  error: 'error',
+  warning: 'warning',
+  success: 'success',
+} satisfies Record<Variant, ActionInfoProps['valueColor']>
 
 type Props = {
   children: ReactNode
-  action?: ReactNode
   className?: string
   isBold?: boolean | null
   isDivider?: boolean
   isMultiLine?: boolean
   label?: ReactNode
   loading?: boolean
   loadingSkeleton?: [number, number]
-  size?: Size
-  textLeft?: boolean
   tooltip?: ReactNode
   variant?: Variant
 }
 
-const DetailInfo = ({
-  action: Action,
+const OldDetailInfo = ({
   className,
   isBold,
   isDivider,
   label,
   loading,
   loadingSkeleton,
-  tooltip: Tooltip,
+  tooltip,
   variant,
   children,
-  size = 'sm',
-  ...props
-}: Props) => {
-  const classNames = `${className} ${isDivider ? 'divider' : ''}`
-
-  return (
-    <Wrapper
-      {...props}
-      size={size}
-      className={classNames}
-      grid
-      gridAutoFlow="column"
-      gridColumnGap={2}
-      isDivider={isDivider}
-      fillWidth
-    >
-      {label && <DetailLabel>{label}</DetailLabel>}
-      <DetailValue haveLabel={!!label} isBold={isBold} variant={variant}>
-        {loading && <Loader skeleton={loadingSkeleton} />}
-        {!loading && (
-          <>
-            {children || '-'} {!!Tooltip && Tooltip} {Action && Action}
-          </>
-        )}
-      </DetailValue>
-    </Wrapper>
-  )
-}
+}: Props) => (
+  <Wrapper
+    className={`${className} ${isDivider ? 'divider' : ''}`}
+    grid
+    gridAutoFlow="column"
+    gridColumnGap={2}
+    isDivider={isDivider}
+    fillWidth
+  >
+    {label && <DetailLabel>{label}</DetailLabel>}
+    <DetailValue haveLabel={!!label} isBold={isBold} variant={variant}>
+      {loading && <Loader skeleton={loadingSkeleton} />}
+      {!loading && (
+        <>
+          {children || '-'} {!!tooltip && tooltip}
+        </>
+      )}
+    </DetailValue>
+  </Wrapper>
+)
+
+const NewDetailInfo = ({
+  className,
+  isBold,
+  isDivider,
+  label,
+  loading,
+  loadingSkeleton,
+  tooltip,
+  variant,
+  children,
+}: Props) => (
+  <>
+    {isDivider && <Divider sx={{ marginBlock: Spacing.sm }} />}
+    <ActionInfo
+      className={className}
+      label={label}
+      value={children || '-'}
+      valueColor={VariantToColorMap[variant || '']}
+      valueTooltip={tooltip}
+      copy={typeof children === 'string'}
```

Comment:
I think this is too general, I don't think there's a need to copy these values

<img width="449" height="79" alt="image" src="https://github.com/user-attachments/assets/a5c5116a-7b11-4b87-860b-27dc96ede887" />
<img width="464" height="88" alt="image" src="https://github.com/user-attachments/assets/baec00cd-a566-4189-889a-996a905dae6f" />

Reply:
Just checked, they indeed did not have buttons

Reply:
Indeed, sorry I'll fix in a follow up PR